.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: monthly
+    groups:
+      actions:
+        patterns:
+          - "*"
```

Comment:
Any reason for the grouping?  Sometimes deps are related and must be updated together, but other times one update is problematic and it will block the others.

Reply:
Mainly to reduce the total number of PRs to reduce "alert fatigue". I and many others get lots of PR/issue notifications, and these multiply on Dependabot days.

Other projects I maintain use grouped updates, and very occasionally one update may be problematic, but like you say, it goes both ways and sometimes they _need_ grouping. 

The vast majority of the time, grouped updates are fine.

Multiple PRs use up more CI time, and are also more cumbersome: after merging one, often it needs to rebase all the others, using more CI time, and needing to wait before merge.